### DISCUSSION TITLE
After last update command list is scrammed on the right

### DISCUSSION THREAD

**Discussion by marekbrze** [LazyVim/LazyVim#5208] [Q&A] [answered]
<img width="1706" alt="Screenshot 2024-12-20 at 10 06 14" src="https://github.com/user-attachments/assets/b2e1b78b-9bdd-481e-846d-562209a59531" />
 I have no idea how to go back for the old look that spans across the bottom... 

![image](https://github.com/user-attachments/assets/abc570f2-9a0c-43c5-bf88-35b4064c9c13)

**Comment by dpetka2001** [comment, marked as answer]
#5159

**Reply by marekbrze** [reply]
Thanks a lot!
